streets_utils/streets_messages/src/deserializers/detected_obj_msg_deserializer.cpp

```diff
@@ -103,7 +103,12 @@ namespace streets_utils::messages::detected_objects_msg {
             std::vector<double> val_row;
 
             for (rapidjson::SizeType  j = 0; j < row.Size(); j++)
-            {
+            {   
+                if ( val[i][j].IsNull() || !val[i][j].IsNumber())
+                {
+                    std::string value = val[i][j].IsNull() ? "null" : val[i][j].GetString();
+                    throw streets_utils::json_utils::json_parse_exception("Covariance matrix contains invalid value" + value + "at position [" + std::to_string(i) + "][" + std::to_string(j) + "]");
```

Comment:
```suggestion
                    throw streets_utils::json_utils::json_parse_exception("Covariance matrix contains invalid value " + value + " at position [" + std::to_string(i) + "][" + std::to_string(j) + "]");
```
Add spaces to improve readability